apps/settings/css/settings.scss

```
@@ -1556,7 +1556,7 @@ doesnotexist:-o-prefocus, .strengthify-wrapper {
 
 			/* various */
 			&#grid-header {
-				@include position('sticky');
+				position: sticky;
```

Maybe this is the correct way of using the `position` function:

```suggestion
				@include position(sticky);
```

---

We are using `position: sticky` in a bunch of other places, so it's probably fine. Alternatively adding ` position: -webkit-sticky;` directly would also work.

With this patch, the grid header in the user's list doesn't scroll anymore and is correctly put at the top.

---

I can replicate

---

Yes, this was for old compatibility, we can drop the mixin.
IE is not supported anymore nor is old versions.

---

> I can't replicate

And I can't anymore too. Maybe it was caused by local changes cached by scsscacher, even after changing branch. Weird 😬

---

We probably should still get rid of this legacy wrappers

---

Safari is handling it just right: https://caniuse.com/?search=position%3A%20sticky

So let's drop it!